modules/ROOT/pages/api-groups-add-sla-tiers.adoc

```diff
@@ -13,18 +13,24 @@ To add an SLA tier:
 . From the left menu, click *SLA Tiers* and then click *Add SLA tier* on the details page.
 +
 .The illustration shows the Add SLA tier window.
-image:api-groups-add-sla-tier.png[Add SLA tier,45%,65%]
+image:api-groups-add-sla-tier.png[Add SLA Tier,45%,65%]
 . Click *Continue*.
 . At the top of the SLA tier details area, click *Default* if you want the SLA tier to apply to API instances in the API Group instance.
 +
 Otherwise, click *Individual* to apply it to a specific API instance within the API Group instance.
++
+*Tip:* You can copy an SLA tier of an API instance, if one exists, when creating an individual limit for that API instance in the API Group instance. You can modify the copied limit, if required:
```

Comment:
consider deleting ", if one exists," as this sentence is a bit long/complicated, and it's unclear whether "one" refers to the tier or the instance--and it goes without saying that you can only copy something that exists?

Reply:
Done! Thanks!